krabs/krabs/testing/extended_data_builder.hpp

```diff
@@ -96,7 +96,7 @@ namespace krabs { namespace testing {
         // No null terminator
         BYTE guid_data[GUID_STRING_LENGTH_NO_BRACES] = {};
 
-        StringFromGUID2(container_id, wide_guid_buffer, GUID_STRING_LENGTH_WITH_BRACES + 1);
+        StringFromGUID2(container_id, wide_guid_buffer, sizeof(wide_guid_buffer));
```

Comment:
This isn't correct. The third parameter should be the count of characters not count of bytes. What symptom were you seeing that made change this?

Reply:
My mistake, rolling back